.gitignore

```diff
@@ -30,3 +30,5 @@ out/
 
 ### VS Code ###
 .vscode/
+/gradle.properties
```

Comment:
@clemensutschig is the `/` at the beginning intended?

Reply:
works :)